calendar_fuzz_test.go

```diff
@@ -1,6 +1,3 @@
-//go:build go1.18
```

Comment:
Why was this removed? The fuzzing feature is newer than the go.mod version.

Reply:
yeah good point, it was when i was updating go version in go.mod, before I realized it would force any consumers to upgrade golang versions.  Reverted that commit

Reply:
Thanks